modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -3,28 +3,38 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
 
 [id="installation-osp-accessing-api-floating_{context}"]
 = Enabling access with floating IP addresses
 
-Make {product-title} API endpoints accessible by attaching two floating IP (FIP) addresses to them: one for the API load balancer (`lb FIP`), and one for {product-title} applications (`apps FIP`).
+Make {product-title} API endpoints accessible by attaching two floating IP (FIP) addresses to them: one for external access to the {product-title} API (`API FIP`), and one for {product-title} applications (`apps FIP`).
```

Comment:
> Make the API endpoints accessible by attaching two FIPs

A couple problems here.

1. here we don't attach the FIPs: here we only create them. We will attach them later, using the Ansible playbooks.
1. only one FIP will be attached to the API servers. The "apps FIP" will be attached to the application servers.

_note: the API is served by the Control Plane (aka "masters"), whereas the applications are served by the Compute nodes (aka "workers")_

Reply:
Got it. Thanks!